libs/perun/components/src/lib/perun-footer/perun-footer.component.ts

```diff
@@ -8,51 +8,29 @@ import { ReportIssueDialogComponent } from '../report-issue-dialog/report-issue-
 import { StoreService } from '@perun-web-apps/perun/services';
 import { getDefaultDialogConfig } from '@perun-web-apps/perun/utils';
 import { TranslateService } from '@ngx-translate/core';
-import { TranslatedElem } from '@perun-web-apps/perun/pipes';
 import { UtilsService } from '@perun-web-apps/perun/openapi';
+import { CopyrightItem, FooterColumn } from '@perun-web-apps/perun/models';
 declare let require: any;
 
-interface FooterElement extends TranslatedElem {
-  logo?: string;
-  icon?: string;
-  dialog?: string;
-  link_en: string;
-}
-
-interface FooterColumn extends TranslatedElem {
-  elements: FooterElement[];
-  logos: string[];
-}
-
-interface FooterCopyrightItems {
-  name: string;
-  url: string;
-}
-
 @Component({
   selector: 'perun-web-apps-footer',
   templateUrl: './perun-footer.component.html',
   styleUrls: ['./perun-footer.component.scss'],
 })
 export class PerunFooterComponent implements OnInit {
-  copyrightTextColor: string = this.storeService.get(
-    'theme',
-    'footer_copyright_text_color'
-  ) as string;
+  copyrightTextColor: string = this.storeService.getProperty('theme').footer_copyright_text_color;
 
   footerColumns: FooterColumn[] = [];
-  copyrightItems: FooterCopyrightItems[] = [];
+  copyrightItems: CopyrightItem[] = [];
   currentYear: number = new Date().getFullYear();
   containsLogos = false;
-  headersTextColor: string = this.storeService.get('theme', 'footer_headers_text_color') as string;
-  linksTextColor: string = this.storeService.get('theme', 'footer_links_text_color') as string;
-  githubRepository: string = this.storeService.get('footer', 'github_releases') as string;
-  githubBackendRepository: string = this.storeService.get(
-    'footer',
-    'github_backend_releases'
-  ) as string;
-  iconColor: string = this.storeService.get('theme', 'footer_icon_color') as string;
-  bgColor: string = this.storeService.get('theme', 'footer_bg_color') as string;
+  headersTextColor: string = this.storeService.getProperty('theme').footer_headers_text_color;
+  linksTextColor: string = this.storeService.getProperty('theme').footer_links_text_color;
+  githubRepository: string = this.storeService.getProperty('footer').github_releases;
+  githubBackendRepository: string = this.storeService.getProperty('footer').github_backend_releases;
+  iconColor: string = this.storeService.getProperty('theme').footer_icon_color;
```

Comment:
I couldn't find any history of the `footer_icon_color` property. Variable `iconColor` is NOT used in HTML, so I think you can delete this line and the property `footer_icon_color` can be deleted from `ConfigProperties.ts`.
```suggestion
```